lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -819,16 +819,63 @@ ChangeResult TypeAnalyzer::visitAtenLinearOp(
   return getLatticeElement(op->getResult(0)).join(knowledge);
 }
 
+// dim_out =
+//  floor((dim_in + 2 * padding - dilation * (kernelSize - 1) - 1) / stride) + 1
```

Comment:
Nit: I don't think this is necessary since this information can very easily be determined from the body of the function